src/components/CopyButton/index.jsx

```diff
@@ -69,6 +69,7 @@ export default class CopyButton extends React.Component {
         onBlur={this.onBlur}
         onMouseUp={this.onMouseUp}
         onMouseDown={this.onMouseDown}
+        data-identifier={this.props['data-identifier']}
```

Comment:
I had to add this data-identifier prop to CopyButton to add it to the underlying HTML button.